modules/serverless-logic-deploying-workflows-dev-mode.adoc

```diff
@@ -102,23 +66,48 @@ $ oc edit sonataflow <workflow_name> -n <your_namespace>
 +
 [source,terminal]
 ----
-$ oc get sonataflows -n <your_namespace>
+$ oc get sonataflows -n <namespace>
+----
+
+.. Find the pod of your application by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n <namespace>
 ----
 
 .. View the workflow logs by running the following command:
 +
 [source,terminal]
 ----
-$ oc logs <workflow_pod_name> -n <your_namespace>
+$ oc logs <workflow_pod_name> -n <namespace>
 ----
 
+.Testing
```

Comment:
Same thought re: non-standard heading for this `.Testing` one. This one feels like splitting it out as its own procedure would be worthwhile (versus including it within another procedure module).

Edit: I see now there already is a ["Testing a deployment"](https://85916--ocpdocs-pr.netlify.app/openshift-serverless/latest/serverless-logic/serverless-logic-getting-started/serverless-logic-deploying-workflows.html#serverless-logic-testing-workflows_serverless-logic-deploying-workflows) procedure later in the assembly. Is that basically the same procedure that's being shown here? If so, could there just be an "Additional resources" link to that procedure from this one instead of duplicating?